File: GXpert/GXpert.Web/Modules/Content/ContentComment/ContentCommentColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace GXpert.Content.Columns;

[ColumnsScript("Content.ContentComment")]
[BasedOnRow(typeof(ContentCommentRow), CheckNames = true)]
public class ContentCommentColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    [EditLink]
    public string CommentText { get; set; }
    public int CommentLeft { get; set; }
    public int CommentRight { get; set; }
    public int TreeId { get; set; }
    public int CommentLevel { get; set; }
    public string ParentCommentText { get; set; }
    public string ContentTitle { get; set; }
    public string Username { get; set; }
    public DateTime InsertDate { get; set; }
    public int InsertUserId { get; set; }
    public DateTime UpdateDate { get; set; }
    public int UpdateUserId { get; set; }
    public short IsActive { get; set; }
}

File: GXpert/GXpert.Web/Modules/ServerTypes/Content/ContentCommentColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { ContentCommentRow } from "./ContentCommentRow";

export interface ContentCommentColumns {
    Id: Column<ContentCommentRow>;
    CommentText: Column<ContentCommentRow>;
    CommentLeft: Column<ContentCommentRow>;
    CommentRight: Column<ContentCommentRow>;
    TreeId: Column<ContentCommentRow>;
    CommentLevel: Column<ContentCommentRow>;
    ParentCommentText: Column<ContentCommentRow>;
    ContentTitle: Column<ContentCommentRow>;
    Username: Column<ContentCommentRow>;
}

export class ContentCommentColumns extends ColumnsBase<ContentCommentRow> {
    static readonly columnsKey = 'Content.ContentComment';
    static readonly Fields = fieldsProxy<ContentCommentColumns>();
}